classes/system/galera/server/database/aodh.yml

```diff
@@ -0,0 +1,15 @@
+parameters:
```

Comment:
How I understand by https://github.com/tcpcloud/salt-formula-mysql/blob/master/mysql/client/server.sls#L4
parameter mysql.server.database is used, not mysql.client.server.database.database from system.mysql.client.database.aodh. But I made it by analogy with mysql formulas of keystone service for the possibility that I missed smth.